runtime/pom.xml

```diff
@@ -19,7 +19,7 @@
         <maven-bundle-plugin.version>5.1.6</maven-bundle-plugin.version>
         <flatten-maven-plugin.version>1.2.7</flatten-maven-plugin.version>
         <!-- java and compiler options -->
-        <java.version>1.8</java.version>
+        <java.version>8</java.version>
```

Comment:
does not work either :-(

Reply:
1.8 should be fine, what is the problem with that?

Reply:
That was my attempt to fix the problem seen in https://github.com/faktorips/faktorips.base/actions/runs/3107664233/jobs/5036022637#step:8:4050:
` Error:  Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project faktorips-valuetypes-joda: Requested toolchain specification did not match any configured toolchain: {version=1.8} -> [Help 1]`

My understanding was that the actions/setup-java would set up the toolchain.

Reply:
> My understanding was that the actions/setup-java would set up the toolchain.

Sadly not see https://github.com/actions/setup-java/issues/276 I'll try to provide a workaround for this in https://github.com/faktorips/faktorips.base/pull/49